hosted_control_planes/hcp-prepare/hcp-enable-disable.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hcp-enable-disable"]
+include::_attributes/common-attributes.adoc[]
+= Enabling or disabling the {hcp} feature
+:context: hcp-enable-disable
+
+toc::[]
+
+The {hcp} feature, as well as the `hypershift-addon` managed cluster add-on, are enabled by default. If you want to disable the feature, or if you disabled it and want to manually enable it, see the following procedures.
+
+include::modules/hcp-enable-manual.adoc[leveloffset=+1]
+include::modules/hcp-enable-manual-addon.adoc[leveloffset=+2]
+
+[id="hcp-disable"]
```

Comment:
Per the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-subsections-level-1-and-lower-headings):

```suggestion
[id="hcp-disable_{context}"]
```